### DISCUSSION TITLE
Vodafone station apertura porta 8123

### DISCUSSION THREAD

**Discussion by Al3eRtO** [indomus/forum#1946] [Networking]
Buongiorno a tutti, 
ho seguito la guida per configurare l'accesso ad Ha dall'esterno, sono bloccato però all'apertura delle porte.
Ho il modem vodafone station sbloccato e lo utilizzo con tim, ho provveduto ad aprire la porta con l'ip del pc dove ho installato Ha senza successo, non riesco a collegarmi dall'esterno e testando con questo sito: https://www.yougetsignal.com/tools/open-ports/ risulta chiusa.
Piccolo ot. il rasberry con cui controllo la stampante 3d funziona senza problemi, ho aperto la relativa porta...
Grazie a tutti!

**Comment by Al3eRtO** [comment]
Nessuno ha qualche idea?

**Reply by drako92** [reply]
Ciao,
in locale come ti colleghi, tramite http://indirizzoip:8123 o https://indirizzoip:8123 ?

**Reply by Al3eRtO** [reply]
Ciao, mi collego tramite https://indirizzoip:8123/

**Reply by Al3eRtO** [reply]
Niente?

**Reply by snakuzzo** [reply]
la regola è semplice...
port forwarding 8123 (TCP) -> 8123 (TCP) verso IP server con HA

**Reply by snakuzzo** [reply]
Ma il problema qual è ?
Vai in timeout o ti dà connection refused?

**Reply by Al3eRtO** [reply]
Siamo d'accordo che la regola sia semplice, ma non funziona. Ho aperto le porte ma facendo il test e provando a collegarmi non funziona
![image](https://user-images.githubusercontent.com/111518919/194359922-77708a0b-081f-42cb-b5e0-7241b4989cb5.png)

**Reply by Al3eRtO** [reply]
Anche con TCP non cambia

**Reply by snakuzzo** [reply]
ma in tutto ciò che c'entra l'UDP ?
Non è che si sceglie il tipo di protocollo a caso. La porta deve essere tassativamente TCP

**Comment by oruamig** [comment]
Ciao,
ho lo stesso problema, ho anche io un modem vodafone sbloccato, navigo  senza problemi però non riesco a entrare su HA da esterno. 
Il mini-pc su cui ho installato HA era raggiungibile da esterno fino a quando è stato collegato a una rete con connessione a fibra, non è stato più raggiungibile da quando l'ho collegaro a un router 4G.

Ho provato con due sim di due gestori differenti senza risultato. 
Sarà una limitazione data dalla rete cellulare?

PS. L'app su cellulare continua comunque a ricevere le notifiche.

**Reply by snakuzzo** [reply]
perché con le reti mobili sei nattato. Hai IP privato e non puoi essere raggiungibile dall'esterno come facevi quando avevi IP pubblico.
In tal caso il consiglio è quello di utilizzare Cloudflared -> Maggiori info [QUI](https://indomus.it/guide/configurare-cloudflare-per-il-proprio-dominio-personale-per-protezione-reverse-proxy-cdn-e-altro/) e [QUI](https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-via-reti-provider-tradizionali-o-che-usino-nat-via-cloudflare/)
